extractor/src/test/java/org/schabi/newpipe/extractor/services/peertube/PeertubeLinkHandlerFactoryTestHelper.java

```diff
@@ -0,0 +1,40 @@
+package org.schabi.newpipe.extractor.services.peertube;
+
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.linkhandler.LinkHandlerFactory;
+import org.schabi.newpipe.extractor.linkhandler.ListLinkHandlerFactory;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class PeertubeLinkHandlerFactoryTestHelper {
+
+    public static void testDoNotAcceptNonURLs(LinkHandlerFactory linkHandler)
```

Comment:
These should be called `assert...` instead of `test...`, but anyway

Reply:
Done: 9ab932e394f31a6687014d69d241dc1a56e4ee18

Reply:
Thanks!